### PR TITLE
Update to share consent with Unity Ads only when user provides an explicit yes/no. In all other cases, Unity Ads SDK will collect its own consent per the guidelines [here][https://unity3d.com/legal/gdpr]

### PR DESCRIPTION
**Pull request by ekrzanich**

### CODE REVIEW

File: UnityAds/MPUnityRouter.m [hunk 1]
```diff
@@ -57,14 +57,21 @@ - (void)initializeWithGameId:(NSString *)gameId
...
 - (void)requestVideoAdWithGameId:(NSString *)gameId placementId:(NSString *)placementId delegate:(id<MPUnityRouterDelegate>)delegate;
 {
-    // Collect and pass the user's consent from MoPub to the Unity Ads SDK
-    UADSMetaData *gdprConsentMetaData = [[UADSMetaData alloc] init];
-    if ([[MoPub sharedInstance] currentConsentStatus] == MPConsentStatusConsented) {
-        [gdprConsentMetaData set:@"gdpr.consent" value:@YES];
-    } else {
-        [gdprConsentMetaData set:@"gdpr.consent" value:@NO];
+    // Collect and pass the user's consent/non-consent from MoPub to the Unity Ads SDK
+    if ([[MoPub sharedInstance] currentConsentStatus] == MPConsentStatusConsented || [[MoPub sharedInstance] currentConsentStatus] == MPConsentStatusDenied) {
```
Comment (rsatrasala) [inline UnityAds/MPUnityRouter.m]: @ekrzanich Do you not want to read the status when "limited ad tracking is enabled"? Do you want to collect the consent yourself in that case?

File: UnityAds/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 
 ## Changelog
+* 2.2.0.6
```
Comment (rsatrasala) [inline UnityAds/CHANGELOG.md]: @ekrzanich Can you review the below to make it more obvious for publishers. "Update to share consent with Unity Ads only when user provides an explicit yes/no. In all other cases, Unity Ads SDK will collect its own consent per the guidelines [here][<relevant Unity Ads documentation>]"?

**Comment by rsatrasala** [comment]
@ekrzanich Changes look good to us. There are two minor comments. If you can review them, we will move forward with the release.

File: UnityAds/CHANGELOG.md (hunk 2 above)
Comment (ekrzanich) [reply UnityAds/CHANGELOG.md]: Update to share consent with Unity Ads only when user provides an explicit yes/no. In all other cases, Unity Ads SDK will collect its own consent per the guidelines [here][https://unity3d.com/legal/gdpr]

File: UnityAds/MPUnityRouter.m (hunk 1 above)
Comment (ekrzanich) [reply UnityAds/MPUnityRouter.m]: We don't handle LAT differently in terms of GDPR

**Pull request merged**